flow-report/test/setup/env-setup.ts

```diff
@@ -4,29 +4,48 @@
  * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
  */
 
-import {jest} from '@jest/globals';
+import fs from 'fs';
+import {MessageChannel} from 'worker_threads';
+
+import jestMock from 'jest-mock';
 import {JSDOM} from 'jsdom';
+import * as preact from 'preact';
 
-/**
- * The jest environment "jsdom" does not work when preact is combined with the report renderer.
- * This sets up our own environment with JSDOM globals.
- */
-beforeEach(() => {
-  const {window} = new JSDOM(undefined, {
-    url: 'file:///Users/example/report.html/',
-  });
-  global.window = window as any;
-  global.document = window.document;
-  global.location = window.location;
-  global.self = global.window;
-
-  // Use JSDOM types as necessary.
-  global.Blob = window.Blob;
-  global.HTMLInputElement = window.HTMLInputElement;
-
-  // Functions not implemented in JSDOM.
-  window.Element.prototype.scrollIntoView = jest.fn();
-  global.self.matchMedia = jest.fn<any, any>(() => ({
-    addListener: jest.fn(),
-  }));
-});
+// @ts-ignore: File is not under 'rootDir'
+import {LH_ROOT} from '../../../root.js';
+
+// https://github.com/testing-library/preact-testing-library/issues/36#issuecomment-1136484478
+fs.writeFileSync(`${LH_ROOT}/node_modules/@testing-library/preact/dist/esm/package.json`,
+  '{"type": "module"}');
+fs.writeFileSync(`${LH_ROOT}/node_modules/@testing-library/preact-hooks/src/package.json`,
```

Comment:
Looks like this was fixed https://github.com/testing-library/preact-testing-library/pull/58

Reply:
It's a similar (same?) issue, but it won't work without (at least, the preact-hooks one? maybe both)